spuc/spuc.py

```diff
@@ -1,25 +1,29 @@
 import click
+import logging
 
 import utils
-from services import google
+from services import gapps_handler, jira_handler
 
 
 class Spuc(object):
-    def __init__(self, config_path, user_config_path):
-        self.config_dict = utils.convert_file_to_yaml(
-                config_path)
-        self.user_config_dict = utils.convert_file_to_yaml(user_config_path)
+    def __init__(self, services_config, user_config):
+        self.user_config = utils.check_is_file_and_convert_from_yaml(
```

Comment:
this doesn't make much sense. you read the config file many times instead of just once.
The spuc class should receive a config file path and read it. after it's read, it should pass on the config dictionary as an argument to all handlers...